js/src/relay/WalletLinkRelay.ts

```diff
@@ -330,7 +331,7 @@ export class WalletLinkRelay implements WalletLinkRelayAbstract {
           this.storage.clear()
           this.ui.reloadUI()
         },
-        err => {
+        (err: string) => {
```

Comment:
Uncertain on this type here. We pass it to the message as a string. This might need to be changed